shared/ui-components/package.json

```diff
@@ -73,9 +73,9 @@
     "react-dom": "^16.13.1",
     "react-focus-lock": "^2.4.1",
     "react-inlinesvg": "^2.0.1",
-    "react-input-mask": "^2.0.4",
     "react-map-gl": "^5.2.7",
     "react-media": "^1.10.0",
+    "react-text-mask": "^5.4.3",
```

Comment:
My main concern with switching is that react-text-mask seems to not be actively maintained anymore. It's still more popular than react-input-mask so I'm not sure it makes us any worse off, but it worries me as a long-term solution. Do we have any better ideas (cc: @jaredcwhite)